packages/mariadb-connection-module/README.md

```diff
@@ -0,0 +1,6 @@
+# `mariadb-connection-module`
+
+> TODO: description
```

Comment:
We have to add some explanation about this module, like the environments that the module use and so on.